src/demo/DemoTestDatabaseService.java

```diff
@@ -49,4 +55,23 @@ private static void testTriggerDatabaseBackup() throws MambuApiException {
 
 		System.out.println("Database request Response:\n" + response);
 	}
+
+	/**
+	 * Tests downloading a DB backup
+	 * 
+	 * @throws MambuApiException
+	 * @throws IOException
+	 */
+	private static void testDownloadLatestDbBackup() throws MambuApiException, IOException {
+
+		methodName = new Object() {}.getClass().getEnclosingMethod().getName();
+		System.out.println(methodName = "\nIn " + methodName);
+
+		DatabaseService databaseService = MambuAPIFactory.getDatabaseService();
+
+		DatabaseBackup response = databaseService.downloadLatestDbBackup();
+
+		System.out.println("Backup was successfuly downloaded = " + (response.getContent().toByteArray() != null));
+
```

Comment:
Thanks for pointing it out, just removed it. We don`t needed to handle IOException I`ve added while testing the content of the DB backup by writing it to a File, and forgot remove it. Just removed it :)